src/main/java/com/github/fishio/PlayerFish.java

```diff
@@ -43,6 +43,10 @@
 
 	private static final double GROWTH_SPEED = 500;
 	private static final double FISH_EAT_THRESHOLD = 1.2;
+	private static final int LIVES = 3;
+	public static final int MAX_LIVES = 5;
```

Comment:
Maybe rename LIVES to START_LIVES or something like that, it's kind of confusing with the other lives : SimpleIntegerProperty attribute.

<naming